src/pmdas/linux/help

```diff
@@ -1318,6 +1318,13 @@ page table entry.
 @ mem.vmstat.nr_zspages number of compressed pages
 @ mem.vmstat.thp_file_alloc count of times huge pages were allocated and put in page cache
 @ mem.vmstat.thp_file_mapped count of times huge pages were used for file mappings
+@ mem.vmstat.pgdemote_direct count of direct reclaims of memory pages
+Counts the direct reclaim and occurs when a page allocation is requested but there is no more space in DRAM.
```

Comment:
@kurik I think this like will cause a QA failure in test 469 which checks for a max help text line width of 80 characters - can you shorten this line?   Thanks!